### DISCUSSION TITLE
Documentation for the different "morph" types

### DISCUSSION THREAD

**Discussion by lsmith77** [explosion/spaCy#12323] [Help: Other Questions] [answered]
Is there any documentation that covers all the different morph types like `Number`, `Case`, `Mood`, `VerbForm` etc and all their possible values?

**Comment by svlandeg** [comment, marked as answer]
Hi! It depends on the pretrained model, but these are typically taken from the "universal dependencies" project: https://universaldependencies.org/u/feat/index.html. That site also describes all the values and their meanings.
